frontend/admin/src/js/components/pool/PoolTable.tsx

```diff
@@ -44,10 +64,21 @@ function viewLinkAccessor(
   editUrlRoot: string,
   id: string,
   title: string | undefined | null,
+  classifications: AccessorClassifications,
+  stream: PoolStream | null,
+  intl: IntlShape,
 ) {
+  let classificationString = ""; // type wrangling the complex type into a string
+  if (classifications && classifications[0]) {
+    const grouping = classifications[0];
+    classificationString = `${grouping.group}-0${grouping.level}`;
+  }
+
   return (
     <Link href={`${editUrlRoot}/${id}`} type="link">
-      {title}
+      {`${title} (${classificationString}${
+        stream ? ` ${intl.formatMessage(getPoolStream(stream))}` : ""
+      })`}
```

Comment:
By default, pools do not have titles so this is causing whitespace before the parentheses. We could likely do something like:

```tsx
{`${title ? `${title} ` : ''}(${classificationString}${stream ? ` ${intl.formatMessage(getPoolStream(stream))}` : ""})`}
```